pkg/apis/provisioning/v1alpha5/constraints.go

```diff
@@ -61,25 +67,28 @@ func (c *Constraints) ToNode() *v1.Node {
 			}
 		}
 	}
+
+	// both the taints and startup taints are applied to nodes we create
+	taints := append(c.Taints, c.StartupTaints...)
```

Comment:
nit: Is there a reason you don't want to inline this with the existing pattern? Is it to have the comment in there?

Reply:
Not sure I follow, on line 83 I need to append again and I don't think this is valid syntax:

```go
taints := append(c.Taints, v1.Taint{Key: NotReadyTaintKey, Effect: v1.TaintEffectNoSchedule}, c.StartupTaints...)
```

Reply:
You're right.